ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Tests/Rollback.hs

```diff
@@ -39,11 +39,13 @@ prop_rollback :: Bool -> QC.Gen QC.Property
 prop_rollback wantRollback = do
   genesisTest <- genChains 1
 
-  let schedule = rollbackSchedule (gtBlockTree genesisTest)
+  let
+      schedulerConfig = noTimeoutsSchedulerConfig (defaultPointScheduleConfig (gtSecurityParam genesisTest))
+      schedule = rollbackSchedule (gtBlockTree genesisTest)
 
   -- | We consider the test case interesting if we want a rollback and we can
   -- actually get one, or if we want no rollback and we cannot actually get one.
-  pure $
+  pure $ withMaxSuccess 10 $
```

Comment:
We really need a policy on this or a better mechanism to have a quick version and a longer version. I'd love to have tests run below 10s total when I develop but have CI test more. 10 tests sounds too little.

Reply:
my view is that as long as we're not working with meaningful properties, we shouldn't bother too much with this…we can worry about it when we've actually added GDD and developed some confidence about what we want to test.

But in general, having different values for CI seems reasonable